### PR TITLE
Leaderboard: final touches

### PR DESCRIPTION
**Pull request by Feliverse**
**Learning objectives**

1. Understand how to use medium-fidelity wireframes to create a UI.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,12 +7,12 @@
...
 
 <body>
-  <h1>Leaderboard</h1>
+  <h1> &#127942 Leaderboard &#127942</h1>
   <main class="contenedor">
     <div class="recentScores">
       <div class="refresh">
         <h3>Recent scores</h3>
-        <button class="btn refresh-button">Refresh</button>
+        <button class="btn refresh-button">&#128472</button>
```
Comment (Qoosim) [inline src/index.html]: - [OPTIONAL] I would recommend you to add `Refresh` text inside that button for better user experience 
your-board

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @Feliverse, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the assessment but you are almost there!

## Top Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct Git Flow. :heavy_check_mark: 



## Suggested changes

- [x] The link to the MIT license in your README file is broken. Kindly follow this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to properly set it up for your project.

license


_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/index.html (hunk 1 above)
Comment (Feliverse) [reply src/index.html]: off course, I will change

Comment (Feliverse) [reply src/index.html]: The LICENCE was created, thank you
image

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @Feliverse , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**